api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -170,131 +178,376 @@ public void test_updateUserWithRetries_register_includes_others() {
   }
 
   @Test
-  public void test_updateUserWithRetries_unregister() {
-    DbUser dbUser = new DbUser();
-    assertThat(userAccessTierDao.findAll()).isEmpty();
-
-    dbUser = userService.updateUserWithRetries(this::unregisterUser, dbUser, Agent.asUser(dbUser));
-
-    // the user has never been registered so they have no DbUserAccessTier entry
-
-    assertThat(userAccessTierDao.findAll()).hasSize(0);
-    Optional<DbUserAccessTier> userAccessMaybe =
-        userAccessTierDao.getByUserAndAccessTier(dbUser, registeredTier);
-    assertThat(userAccessMaybe).isEmpty();
-  }
+  public void testSimulateUserFlowThroughRenewal() {
```

Comment:
@petesantos I modified these a bit.  Please confirm that they are still checking what you intended.

Reply:
looks good - I wanted to have a test that mirrored what the flow would look like in the wild